### ISSUE TITLE
Update plugin with register and update?

### ISSUE THREAD

**Issue by fbacker** [DriveWealth/react-native-onepassword#6]
Any plan to update the plugin?

Missing features as register and update password
https://github.com/AgileBits/onepassword-app-extension

cheers

**Comment by jjshammas** [comment]
Yes, would love to add those other methods but have not gotten around to it yet. PRs always appreciated!
